app/src/main/java/com/eveningoutpost/dexdrip/LibreReceiver.java

```diff
@@ -146,11 +153,33 @@ private static void saveSensorStartTime(Bundle sensor, String serial) {
     private static double calculateWeightedAverage(List<Libre2RawValue> rawValues, long now) {
         double sum = 0;
         double weightSum = 0;
+        DecimalFormat longformat = new DecimalFormat( "#,###,###,##0.00" );
+
+        libre_calc_doku="";
         for (Libre2RawValue rawValue : rawValues) {
             double weight = 1 - ((now - rawValue.timestamp) / (double) SMOOTHING_DURATION);
             sum += rawValue.glucose * weight;
             weightSum += weight;
-        }
+            libre_calc_doku += DateFormat.format("kk:mm:ss :",rawValue.timestamp) + " w:" + longformat.format(weight) +" raw: " + rawValue.glucose  + "\n" ;
+           }
         return Math.round(sum / weightSum);
     }
+
+
+    public static List<StatusItem> megaStatus() {
+        final List<StatusItem> l = new ArrayList<>();
+
+        l.add(new StatusItem("Libre2 Sensor",Sensor.currentSensor().uuid + "\nStart: " +  DateFormat.format("dd.MM.yyyy kk:mm",Sensor.currentSensor().started_at) ));
+        String lastReading ="";
+        try {
+            lastReading = DateFormat.format("dd.MM.yyyy kk:mm:ss", last_reading).toString();
+            l.add(new StatusItem("Last Reading", lastReading));
+        } catch (Exception e) {
+            Log.e(TAG, "Error readlast: " + e);
+        }
+        l.add(new StatusItem("Last Calc.",libre_calc_doku));
```

Comment:
Last calc looks like so much technical information it should be hidden behind engineering mode. I'm not sure whether the Sensors section would want the same or not.

Reply:
The lastcalc could be a good idea to put it to engenieering mode. Will do that the evening.

For the Sensors I think this is a valuable information for everyone of the users, to see if the Sensors go often in a bad state, and also to have the Serial-Number in one Place. What do you think to add this as switchable in preferences to make it easy accessible for a normal user, but standard put to off?